dashboard/legacy/middleware/helpers/library_bucket.rb

```diff
@@ -7,7 +7,9 @@ def initialize
   end
 
   def allowed_file_types
-    %w(.json .java .py .csv .txt .js)
+    # The Library bucket is used by App Lab (json), Java Lab (java, csv, txt),
+    # Python Lab (py, csv, txt), and Web Lab 2 (html, css, js, md).
```

Comment:
`weblab2` doesn't support `md` files yet. Could add at https://github.com/code-dot-org/code-dot-org/blob/230347305df026996742f4f5c50a5a64c287f31f/apps/src/weblab2/constants.ts#L1.

Reply:
Yep I'm doing that as a follow-up, tracked [here](https://codedotorg.atlassian.net/browse/AFL-180). I just made the change in advance here since I was already touching the backpack list.